img/browser.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="76px" height="59px" viewBox="0 0 76 59" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
+    <title>browser</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="dribbble" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
I imagine it would only be an issue when attempting point to the ids differently, though I am not an svg expert. Though I can at very least update the ids to be different.